packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -3922,6 +4001,20 @@ class _MenuDefaultsM3 extends MenuStyle {
 
   @override
   VisualDensity get visualDensity => Theme.of(context).visualDensity;
+
+  @override
+  AnimationStyle get sizeAnimationStyle {
+    return switch (Theme.of(context).platform) {
+      TargetPlatform.iOS || TargetPlatform.android || TargetPlatform.fuchsia
+        => AnimationStyle(
+          curve: Curves.easeInOutCubicEmphasized,
+          reverseCurve: Curves.easeInOutCubicEmphasized.flipped,
+          duration: const Duration(milliseconds: 500),
```

Comment:
Why is there another 500ms duration? How does this work with the default animation controller in `_MenuAnchorState`?

Reply:
This is the class to define all the defaults from tokens. I've removed the other one! Thanks for catching this!